src/view/actions/sendEvent.jsx

```diff
@@ -84,14 +85,8 @@ const getSettings = ({ values }) => {
 };
 
 const validationSchema = object().shape({
-  xdm: string().matches(
-    singleDataElementRegex,
-    "Please specify a data element"
```

Comment:
I moved this text into a constant since we use it so much and so it will stay consistent.